extension/src/popup/views/GrantAccess.tsx

```diff
@@ -31,14 +33,21 @@ const SubheaderEl = styled.h3`
   letter-spacing: 0.1px;
   color: ${COLOR_PALETTE.primary}};
 `;
+const PublicKeyEl = styled.p`
+  color: ${COLOR_PALETTE.primary}};
+`;
 const ButtonContainerEl = styled.div`
   display: flex;
   justify-content: space-around;
-  padding: 3rem 1.25rem;
+  padding: 3rem 0;
   flex-direction: row;
 `;
 const RejectButtonEl = styled(Button)`
   background: ${COLOR_PALETTE.text};
+  width: 9.75rem;
+`;
+const SharePublicKeyButtonEl = styled(SubmitButton)`
+  width: 12.25rem;
```

Comment:
While I was in here, I addressed some changes that were made in the latest round of designs